README.md

```diff
@@ -23,7 +23,7 @@ info: Server running at http://localhost:8080
 
 ```bash
 $ vim ./local.yaml # See below for configuration
-$ docker run --rm -it --volume=`pwd`/local.yaml:/config/local.yaml -p 8080 screwdrivercd/api:latest
+$ docker run --rm -it --volume=`pwd`/local.yaml:/config/local.yaml -p 8080 screwdrivercd/screwdriver:latest
```

Comment:
Amended.

Last time we updated the `stable` tags, it was manually done. We should re-visit our policy on how we treat how we tag docker images, especially if that's going to be a way users interact with the system.